src/cli/api/local_api.py

```diff
@@ -36,24 +54,140 @@ def __is_config_valid(self, application: str) -> bool:
+        application_json_path = app_config_path / 'application.json'
+        network_json_path = app_config_path / 'network.json'
+
+        application_data = read_json_file(application_json_path)
+        network_data = read_json_file(network_json_path)
+
+        app_config = {"application": application_data, "network": network_data}
+        return app_config
 
     def create_experiment(
-        self, name: str, app_config: AppConfigType, local: bool
-    ) -> None:
-        pass
+        self, name: str, app_config: AppConfigType, network: str, path: Path, application: str
+    ) -> Tuple[bool, str]:
 
-    def delete_experiment(self, path: str) -> None:
-        pass
+        experiment_directory = path / name
+        if experiment_directory.is_dir():
+            return False, f'Experiment directory {name} already exists.'
+
+        experiment_directory.mkdir(parents=True)
+
+        input_directory = experiment_directory / 'input'
+        input_directory.mkdir(parents=True)
+        self.__copy_input_files_from_application(application, input_directory)
+
+        experiment_json_file = experiment_directory / 'experiment.json'
+        experiment_meta = {
+            "backend": {
+                "location": "local",
+                "type": "local_netsquid"
+             },
+            "number_of_rounds": 1,
+            "description": f"{name}: experiment description"
+        }
+
+        asset_application = self.__create_asset_application(app_config)
+        asset_network = self.__create_asset_network(network, app_config)
+        asset = {"network": asset_network, "application": asset_application}
+
+        experiment_data = {'meta': experiment_meta, 'asset': asset}
+        write_json_file(experiment_json_file, experiment_data)
+
+        return True, "Success"
 
-    def run_experiment(self, path: str, block: bool) -> Optional[List[ResultType]]:
+    def __create_asset_application(self,  app_config: AppConfigType) -> List[Dict[str, Any]]:
+        return []
+
+    def __create_asset_network(self,  network: str, app_config: AppConfigType) ->  Dict[str, Any]:
+        return {}
+
+    def __copy_input_files_from_application(self,  application: str, input_directory: Path) -> None:
+        """
+        Copy the input/source files of the 'application' to the 'input_directory'
+
+        Args:
+            application: The application name for which the input files need to be copied
+            input_directory: The destination where application files need to be stored
+
+        """
+
+
+    def check_valid_network(self, network: str, app_config: AppConfigType) -> bool:
+        """
+        Check if the network name is a valid network for the Application
+
+        Args:
+            network: The network name
+            app_config: Application Configuration containing the available networks
+
+        Returns:
+            bool: True if the given network name is available in application configuration, False otherwise
```

Comment:
I think it is a valid network for the app when the number of nodes of the network > number of roles for the app
Second I doubt the input parameter app_config. In app_config we already have the network field with the possible networks? What do we need to check here